inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/model/Predictions.java

```diff
@@ -166,12 +166,9 @@ public Predictions(Project aProject, User aUser) {
         Type type = CasUtil.getType(aJcas.getCas(), aLayer.getName());
         List<AnnotationFS> existingAnnotations = CasUtil.selectCovered(aJcas.getCas(), 
                 type, aWindowBegin, aWindowEnd);
-        List<Integer> existingOffsets = existingAnnotations.stream().map(e -> e.getBegin())
-                .collect(Collectors.toList());
 
         return predictions.entrySet().stream()
                 .filter(f -> f.getKey().getDocumentName().equals(aDocumentName))
-                .filter(f -> !existingOffsets.contains(f.getKey().getOffset().getBeginCharacter()))
```

Comment:
Same as commented out overlapping code before: handling overlapping annotations should be restored before merging the PR.